pt-BR/testing.md

```diff
@@ -23,51 +23,46 @@ Ao rodar seus testes no Rails você pode garantir que seu código continua com a
 
 Os testes no Rails podem simular requisições no browser e com isso, você pode testar a resposta da sua aplicação sem ter que testar utilizando de fato seu o browser.
 
-Introduction to Testing
+Introdução a testes
 -----------------------
 
-Testing support was woven into the Rails fabric from the beginning. It wasn't an "oh! let's bolt on support for running tests because they're new and cool" epiphany.
+Suporte a testes foi implantado no Rails desde os primórdios. Não foi uma epifania do tipo: "Vamos adicionar suporte para testes porque eles são novos e legais!"
 
-### Rails Sets up for Testing from the Word Go
+### Configurações para testes em aplicações Rails
 
-Rails creates a `test` directory for you as soon as you create a Rails project using `rails new` _application_name_. If you list the contents of this directory then you shall see:
+Rails cria um diretório `test` para você logo que você cria um projeto Rails usando o comando `rails new` _nome_da_aplicacao_. Se você deseja listar o conteúdo deste diretório, então você vai ver:
 
 ```bash
 $ ls -F test
 application_system_test_case.rb  controllers/                     helpers/                         mailers/                         system/
 channels/                        fixtures/                        integration/                     models/                          test_helper.rb
 ```
 
-The `helpers`, `mailers`, and `models` directories are meant to hold tests for view helpers, mailers, and models, respectively. The `channels` directory is meant to hold tests for Action Cable connection and channels. The `controllers` directory is meant to hold tests for controllers, routes, and views. The `integration` directory is meant to hold tests for interactions between controllers.
+Os diretórios `helpers`, `mailers` e `models` são destinados a realizar os testes para view helpers, mailers e models, respectivamente. O diretório `channel` é destinado a realizar os testes para a conexão e canais do ActionCable. O diretório `controllers` se destina a realizar testes para os controllers, rotas e views. O diretório `integration` se destina a realizar testes entre a interação entre controllers.
```

Comment:
```suggestion
Os diretórios `helpers`, `mailers` e `models` são destinados a realizar os testes para *view helpers*, *mailers* e *models*, respectivamente. O diretório `channel` é destinado a realizar os testes para a conexão e canais do *ActionCable*. O diretório `controllers` se destina a realizar testes para os *controllers*, rotas e *views*. O diretório `integration` se destina a realizar testes entre a interação entre *controllers*.
```

Acho que aqui seria legal ter alguns itálicos nos estrangerismos o que acha?

Reply:
Excelente sugestão